playbooks/roles/katello/tasks/module_pr.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: 'install yard gem'
+  shell: >
+      /opt/puppetlabs/puppet/bin/gem install yard
+  when: puppet_repositories_version == 4
+
+- name: 'install puppetlabs-strings'
+  shell: >
+      /opt/puppetlabs/puppet/bin/puppet module install puppetlabs-strings
+  when: puppet_repositories_version == 4
+
+- name: "install module PR into installer directory"
+  shell: >
+      cd {{ (item.split("/")[0] == 'theforeman') | ternary("/usr/share/foreman-installer/modules", "/usr/share/katello-installer-base/modules") }} &&
+      rm -rf {{ item.split("/")[1] }} &&
+      git clone https://github.com/{{ item.split("/")[0] }}/puppet-{{ item.split("/")[1] }} {{ item.split("/")[1] }} &&
+      cd {{ item.split("/")[1] }} &&
+      git fetch origin pull/{{ item.split("/")[2] }}/head:pr &&
+      git merge pr
```

Comment:
It is intentionally this way - the particular PR branch could be out of date and missing something  important, and it reflects the reality of what will happen the PR is actually merged.